diff --git a/webprofiler/src/DataCollector/ThemeDataCollector.php b/webprofiler/src/DataCollector/ThemeDataCollector.php
index 5b1116b..c16b8cd 100644
--- a/webprofiler/src/DataCollector/ThemeDataCollector.php
+++ b/webprofiler/src/DataCollector/ThemeDataCollector.php
@@ -64,7 +64,7 @@ class ThemeDataCollector extends DataCollector implements DrupalDataCollectorInt
       'owner' => $activeTheme->getOwner(),
       'baseThemes' => $activeTheme->getBaseThemeExtensions(),
       'extension' => $activeTheme->getExtension(),
-      'styleSheetsRemove' => $activeTheme->styleSheetsRemove(),
+      'styleSheetsRemove' => method_exists($activeTheme, 'styleSheetsRemove') ? $activeTheme->styleSheetsRemove() : $activeTheme->getStyleSheetsRemove(),
       'libraries' => $activeTheme->getLibraries(),
       'regions' => $activeTheme->getRegions(),
     ];
